ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/InMemory.hs

```diff
@@ -101,6 +101,7 @@ import           Ouroboros.Network.AnchoredSeq (Anchorable (..),
 import qualified Ouroboros.Network.AnchoredSeq as AS
 
 import           Control.Exception
+import           Data.Coerce (coerce)
 import           Ouroboros.Consensus.Block
```

Comment:
```suggestion
import           Control.Exception
import           Data.Coerce (coerce)

import           Ouroboros.Consensus.Block
```

Perhaps keep the separation. See point 15 in https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus/docs/StyleGuide.md#formatting